docker-build/action.yaml

```diff
@@ -111,7 +117,6 @@ runs:
           ${{ inputs.build-args }}
         push: ${{ inputs.push }}
         load: ${{ inputs.load }}
-        tags: ${{ steps.config.outputs.image-name }}:${{ inputs.image-version }}
+        tags: ${{ steps.config.outputs.image-name }}:${{ inputs.image-version || github.sha }}
```

Comment:
Looking above, image-version is a required parameter. It feels like this shouldn't happen? Or instead set it to non required and default it to `github.sha`?